Problems/P.000025

```diff
@@ -8,9 +8,10 @@ $\R^3$?
 * Status/Conjectures: Open.
 * Partial and Related Results:
 It is NP-hard to obtain the minimum-facet surface
-separating two nested convex polytopes~\cite{dg-ocop3-97a},
-but
-polynomial-time approximation algorithms are known
+separating two nested convex polytopes~\cite{dg-ocop3-97a}, if they are allowed to have distinct structures; 
+it is unknown if it is hard when one polytope is an offset of the other, e.g., if they arise as offsets of an underlying convex polytope
+that we seek to approximate.
+Polynomial-time approximation algorithms are known
 (\cite{bg-aoscf-95,ms-sapo-95,as-sagp-98})
 for this case, and for separating two terrain surfaces,
```

Comment:
I worry that your change above makes the meaning of "for this case" here unclear. Ideas for how to fix this?

Reply:
Replace "for this case" by "for the case of nested convex polytopes"?

Reply:
OK, I took a stab at revising. Can you take a look and confirm before I merge?